tests/vmi_configuration_test.go

```diff
@@ -2697,7 +2697,7 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 
 				cpuVmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU] = resource.MustParse("2")
 				Vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU] = resource.MustParse("1")
-				Vmi.Spec.NodeSelector = map[string]string{v1.CPUManager: "true"}
+				Vmi.Spec.NodeSelector = map[string]string{v1.DeprecatedCPUManager: "true"}
```

Comment:
I don't understand the need to set this on the VMI, surely `DedicatedCPUPlacement` being true above is enough?

Reply:
I don't understand as well but I just refactor the variable which caused the change here.